src/main/java/venture/Aust1n46/chat/api/events/PrivateMessageEvent.java

```diff
@@ -0,0 +1,49 @@
+package venture.Aust1n46.chat.api.events;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import venture.Aust1n46.chat.model.VentureChatPlayer;
+
+/**
+ * Event called when a private message has been sent to a player.
+ * This event can not be cancelled.
+ *
+ * @author LOOHP
+ */
+@Getter
+public class PrivateMessageEvent extends Event {
+
+    private static final HandlerList handlers = new HandlerList();
+
+    private final VentureChatPlayer sender;
+    private final VentureChatPlayer receiver;
+    private final String msg;
+    private @Setter String echo;
+    private @Setter String send;
+    private @Setter String spy;
+    private final boolean bungee;
+
+    public PrivateMessageEvent(VentureChatPlayer sender, VentureChatPlayer receiver, String msg, String echo, String send, String spy, boolean bungee) {
+        super(!Bukkit.isPrimaryThread());
```

Comment:
Would be preferable to pass this in if possible rather than making a static call.  Hurts future testability.